src/utils/async.ts

```diff
@@ -27,7 +37,7 @@ export const mapAsyncIterable = <T, V>(
   const iterMapped = mapAsyncIterator(iter, f);
   return {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    [Symbol.asyncIterator]: (): AsyncIterator<V, any, undefined> => iterMapped
+    [Symbol.asyncIterator]: (): AsyncIterator<V, V, undefined> => iterMapped
```

Comment:
Should we change the functions signature as well?

Reply:
Yes, I think it should be coherent with the new override.

Reply:
ok, I'll do it

Reply:
Did it in https://github.com/pagopa/io-functions-commons/pull/198/commits/62be48c3360017e181d9e05a0c44d16acf2a27e8. 

However, I wonder we should do the same in https://github.com/pagopa/io-functions-commons/blob/62be48c3360017e181d9e05a0c44d16acf2a27e8/src/utils/async.ts#L148

Reply:
Yes, better do it. Dit it in https://github.com/pagopa/io-functions-commons/pull/198/commits/af35e21cc6e74518e914899a43ddc4eed87fd8b6.